source/blog/2010-05-13-the-switch-to-git.md

```diff
@@ -23,7 +23,7 @@ do so by using the read-only SVN features of github.com:
 -   Doctrine 1.2 -
     [http://svn.github.com/doctrine/doctrine1.git](http://svn.github.com/doctrine/doctrine1.git)
 -   Doctrine 2.0 -
-    [http://svn.github.com/doctrine/doctrine2.git](http://svn.github.com/doctrine/doctrine2.git)
+    [http://svn.github.com/doctrine/orm.git](http://svn.github.com/doctrine/orm.git)
```

Comment:
404. The old link must be broken too when the old name was valid.

@jwage Is SVN still a thing?

Reply:
so just delete it?

Reply:
I would just leave it for historical purposes.

Reply:
Which would result in PRs or issues that addresses the 404 errors.